client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -37,11 +42,8 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 					</div>
 				) }
 				<p className={ `launchpad__checklist-item-text` }>{ title }</p>
-				{ task.displayBadge && task.badgeText ? (
-					<Badge type="info-blue">{ task.badgeText }</Badge>
-				) : null }
-				{ /* don't display chevron and badge at the same time */ }
-				{ ! taskDisabled && ! task.displayBadge && ! task.badgeText && (
```

Comment:
I notice here we check both `displayBadge` and `badgeText`, where in the replacement above we got rid of checking `displayBadge` altogether. Would we want to include that there as well?

Side note im wondering why we have both... should `badgeText` being truthy imply `displayBadge`? Or is it beneficial to have both?

Reply:
I've added the changes on line 44 and for the sake of consistency with line 40 which was added before I made sure to check for both `displayBadge` and `badgeText`. I think we agreed with Anthony that there shouldn't be a case where one would be truthy and the other falsy so I think it's fine to just check for `badgeText`

Reply:
If that's true, should consider deleting displayBadge as a task property entirely? If we're not using it for conditionals, seems like we wouldn't need it? If we decide to do that, it could (maybe should) be done in a separate PR.

Reply:
> seems like we wouldn't need it? If we decide to do that, it could (maybe should) be done in a separate PR.

That seems to make sense to me as well. I agree with the follow-up if others are on board with the idea.